app/forms/pages/type_of_answer_form.rb

```diff
@@ -4,15 +4,32 @@ class Pages::TypeOfAnswerForm < BaseForm
   validates :draft_question, presence: true
   validates :answer_type, presence: true, inclusion: { in: Page::ANSWER_TYPES }
 
-  def submit(session)
+  def submit
     return false if invalid?
+    return true unless answer_type_changed?
```

Comment:
We did have a similar guard for "Edit question" journey but it wasn't clear what would happen if someone managed to call this method directly instead of through a controlled route.